README.md

```diff
@@ -23,17 +23,13 @@ Plugins for a Unity project use the following folder structure:
 
 ## Installation
 
-#### Building the ACPCore Unity Package
-1. Clone the [ACPCore Unity repository](https://github.com/adobe/unity-acpcore).
-2. In the root directory, run the following command to build the ACPCore Unity package
-    `make release`
-3. Import `ACPCore.unitypackage` via Assets-Import Package from `bin -> build_temp -> ACPCore-1.0.0-Unity -> ACPCore` folder.
-
-#### Building the ACPUserProfile Unity Package
-1. Clone the current repository.
-2. In the root directory, run the following command to build the ACPUserProfile Unity package
-    `make release`
-3. Import `ACPUserProfile.unitypackage` via Assets-Import Package from `bin -> build_temp -> ACPUserProfile-1.0.0-Unity -> ACPUserProfile` folder.
+- Download [ACPCore-1.0.0-Unity.zip](https://github.com/adobe/unity-acpcore/tree/master/bin/ACPCore-1.0.0-Unity.zip) 
```

Comment:
This should be 1.0.1, I think we can actually point to the bin folder instead of the actual zip, so in case the ACPCore gets updated in the future these links will still be valid.

Reply:
Ok, let's use ACPCore-1.0.1-Unity.zip with link: https://github.com/adobe/unity-acpcore/tree/master/bin/ACPCore-1.0.1-Unity.zip